tests/integration/suite/daprd/pubsub/pubsub.go

```diff
@@ -16,6 +16,7 @@ package pubsub
 import (
 	_ "github.com/dapr/dapr/tests/integration/suite/daprd/pubsub/contentlength"
 	_ "github.com/dapr/dapr/tests/integration/suite/daprd/pubsub/grpc"
+	_ "github.com/dapr/dapr/tests/integration/suite/daprd/pubsub/grpc/traceparent"
 	_ "github.com/dapr/dapr/tests/integration/suite/daprd/pubsub/http"
```

Comment:
Can you add an IT test for both http && bulk pubsub?

```suggestion
	_ "github.com/dapr/dapr/tests/integration/suite/daprd/pubsub/http"
	_ "github.com/dapr/dapr/tests/integration/suite/daprd/pubsub/http/traceparent"
```

An idea: you could add a separate `t.Run` in the `basic.go` for both grpc + http for `bulk pubsub` as well so you dont need a new file for bulk pubsub to be tested.